cmd/replay/main.go

```diff
@@ -198,11 +200,23 @@ func main() {
 		}
 		genesisBlockNum, err := initialArbosState.GenesisBlockNum()
 		if err != nil {
-			panic(fmt.Sprintf("Error getting chain ID from initial ArbOS state: %v", err.Error()))
+			panic(fmt.Sprintf("Error getting genesis block number from initial ArbOS state: %v", err.Error()))
 		}
-		chainConfig, err := arbos.GetChainConfig(chainId, genesisBlockNum)
+		chainConfigJson, err := initialArbosState.ChainConfig()
 		if err != nil {
-			panic(err)
+			panic(fmt.Sprintf("Error getting chain config from initial ArbOS state: %v", err.Error()))
+		}
+		var chainConfig params.ChainConfig
+		err = json.Unmarshal(chainConfigJson, &chainConfig)
```

Comment:
If `len(chainConfigJson) == 0`, we should fall back to the old way of `arbos.GetChainConfig`